user-service/utils/utils.go

```diff
@@ -107,3 +107,29 @@ func SendEmail(newPw string, emailTo string) {
 		panic(err)
 	}
 }
+
+func GetAuthBearerToken(c *fiber.Ctx) (string, error) {
+
+	// Get the Authorization header from the request
+	authHeader := c.Get("Authorization")
+
+	// Check if the Authorization header is empty
+	if authHeader == "" {
+		c.Status(fiber.StatusUnauthorized)
+		return "", c.JSON(fiber.Map{
+			"message": "Missing token",
+		})
+	}
+
+	// Split the Authorization header into two parts: "Bearer" and the token
+	parts := strings.Split(authHeader, " ")
+	if len(parts) != 2 || parts[0] != "Bearer" {
+		c.Status(fiber.StatusUnauthorized)
```

Comment:
Going forward, we could consider decoupling sending responses from actually creating the JSON structs to make testing easier. Works for now!